db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/CastNode.java

```diff
@@ -1101,6 +1123,7 @@ private void genDataValueConversion(ExpressionClassBuilder acb,
              * of VSDV.
              */
 
+            // not sure if this is important
```

Comment:
@martinrupp are you familiar with that? I seem to remember that you worked with precisions before.